src/commands/install-php.yml

```diff
@@ -0,0 +1,22 @@
+description: "Install PHP into your Ubuntu or Debian based image."
+parameters:
+  version:
+    description: "The PHP version."
+    type: string
+    default: "7.3"
```

Comment:
Let's bump this to 7.4, since it's the latest release.

Reply:
done in https://github.com/CircleCI-Public/php-orb/pull/25/commits/3d2eccf9ee053941036864c41c41cd2018085674